proto/one/enterprise/report/report_definition.proto

```diff
@@ -37,11 +37,18 @@ message ReportDefinition{
     // When it was last run
     JsonTicksDateTime lastRunOn = 9;
 
+    // Version
+    int32 version = 10;
+
+    // The Report Definition owned plant belonging Tenant Id
+    google.protobuf.StringValue tenantId = 11;
```

Comment:
I think the google wrapper isn't needed as you can assign null to strings by default in C#

Reply:
@JonathanRobertson-AQI , it is better to be nullable since the new Version and TenantId are server internal used properties, ideally we should add [JsonIgnore] avoid it returned back to browser side via applicaiton/json content type response, however, not sure how to do that. 
So if web client has this field definition, the post/put/patch value will be discard and being set from Report micro service.